src/sentry/constants.py

```diff
@@ -695,8 +695,13 @@ def from_str(cls, string: str) -> int:
 HEALTH_CHECK_GLOBS = [
     "*healthcheck*",
     "*healthy*",
```

Comment:
As mention in https://github.com/getsentry/sentry/issues/58219#issuecomment-1766074573 I think we should tighten `healthy` too.